osu.Game.Tests/Resources/storyboard_no_video.osu

```diff
@@ -0,0 +1,487 @@
+osu file format v14
+
+[General]
+AudioFilename: Namae no nai Kaibutsu -  EGOIST.mp3
+AudioLeadIn: 1500
+PreviewTime: 60090
+Countdown: 0
+SampleSet: Soft
+StackLeniency: 0.4
+Mode: 0
+LetterboxInBreaks: 0
+WidescreenStoryboard: 1
+
+[Editor]
+DistanceSpacing: 1
+BeatDivisor: 4
+GridSize: 32
+TimelineZoom: 1.7
+
+[Metadata]
+Title:Namae no Nai Kaibutsu (TV Size)
+TitleUnicode:名前のない怪物  (TV Size)
+Artist:EGOIST
+ArtistUnicode:EGOIST
+Creator:Nevo
+Version:Desire
+Source:PSYCHO-PASS
+Tags:japanese lasse frostwich synderes ggf gotogofast stack lince irohas yukiyo Gottagof4st chekito
+BeatmapID:2157090
+BeatmapSetID:1031560
+
+[Difficulty]
+HPDrainRate:6
+CircleSize:4
+OverallDifficulty:9
+ApproachRate:9
+SliderMultiplier:1.7
+SliderTickRate:1
+
+[Events]
+//Background and Video events
+0,0,"BG.jpg",0,0
+Video,0,"nevo cute.avi"
```

Comment:
We try to not copy full beatmaps for copyright reasons. Most of the contents of this .osu can be culled (just keep two hitobjects or something + the video). Likewise, remove the things above (name/creator/artist/etc).